### DISCUSSION TITLE
Loading placeholder not showing in my admin panel.

### DISCUSSION THREAD

**Discussion by amazingakai** [piccolo-orm/piccolo_admin#388] [Q&A]
I just found out that there's a loading placeholder in piccolo admin on the demo admin panel.

It's not showing up in my admin panel for some reason though, is it a bug or what?

[Screencast_20240501_203126.webm](https://github.com/piccolo-orm/piccolo_admin/assets/93782497/179d7af1-132e-44cb-af9c-53ae5171ca3f)

**Comment by dantownsend** [comment]
I think the demo is running an older version - I don't automatically update it on each release.

The loading indicator might have broken after we transitioned to Vue 3. We should try and add it back if it's an easy fix.

**Reply by amazingakai** [reply]
Ah I see, thanks!

**Reply by sinisaos** [reply]
@AmazingAkai Yes, you are right, the loading indicator is not working. Something similar to the previous loading indicator can be achieved by moving this piece of code 

https://github.com/piccolo-orm/piccolo_admin/blob/5e8e96d56656cbf7283b63f8729d0748eb895867/admin_ui/src/views/RowListing.vue#L81-L85

outside the `template`  tag like this

```javascript
<template>
    <BaseView>
        <transition name="fade">
            <p
                v-show="loadingStatus"
                id="loading_indicator"
                style="padding-left: 1rem"
            >
                {{ $t("Loading") }} ...
            </p>
        </transition>
        <template v-if="!loading">
       ... 
```
Result is this:

[loading_indicator.webm](https://github.com/piccolo-orm/piccolo_admin/assets/30960668/77b787fb-7df3-4b31-a110-86eae77ac6f4)
